lib/foreman_openscap/engine.rb

```diff
@@ -7,13 +7,6 @@ class Engine < ::Rails::Engine
     engine_name 'foreman_openscap'
     config.autoload_paths += Dir["#{config.root}/app/controllers/concerns"]
     config.autoload_paths += Dir["#{config.root}/app/helpers/concerns"]
```

Comment:
Why do these 2 remain, but you remove the others?

Reply:
I think there is no reason to leave them, removed.